src/LinuxTracing/ThreadStateManagerTest.cpp

```diff
@@ -19,17 +20,21 @@ using orbit_grpc_protos::ThreadStateSlice;
 
 TEST(ThreadStateManager, OneThread) {
   constexpr pid_t kTid = 42;
+  constexpr pid_t kWTid = 420;
+  constexpr pid_t kWPid = 4200;
```

Comment:
I think these variable names should be longer. Otherwise it's a bit too unclear what they mean. What about `kWasBlockedByPid` and `kWasBlockedByTid`?